package.json

```diff
@@ -1,22 +1,110 @@
+    "angular-resource": "1.4.4",
+    "angular-route": "1.4.4",
+    "angular-sanitize": "1.4.4",
+    "angular-ui-bootstrap": "^2.5.0",
+    "autotrack": "^2.4.1",
+    "backbone": "0.9.1",
+    "backbone-1.3.2": "npm:backbone#1.3.2",
+    "backbone.marionette": "2.4.5",
+    "blazy": "1.6.2",
+    "bootstrap": "3.4.1",
+    "bootstrap-daterangepicker": "3.0.3",
+    "bootstrap-switch": "3.3.2",
+    "bootstrap-timepicker": "0.5.1",
+    "bootstrap3-typeahead": "bassjobsen/Bootstrap-3-Typeahead#~3.1.1",
+    "clipboard": "1.5.15",
+    "css-grid-polyfill-binaries": "FremyCompany/css-grid-polyfill-binaries#1.1.1",
+    "d3": "3.5.17",
+    "datamaps": "^0.5.8",
+    "datatables": "1.10.9",
+    "datatables-bootstrap3": "Jowin/Datatables-Bootstrap3",
+    "datatables-buttons": "DataTables/Buttons#^1.5.6",
+    "datatables-colreorder": "DataTables/ColReorder#^1.5.1",
+    "datatables-fixedcolumns": "DataTables/FixedColumns#3.2.0",
+    "datatables-fixedheader": "DataTables/FixedHeader#^3.1.3",
+    "datatables-scroller": "DataTables/Scroller#^1.5.1",
+    "datetimepicker": "npm:jquery-datetimepicker#2.5.4",
     "eslint-plugin-eslint-dimagi": "^1.0.0",
-    "js-xpath": "dimagi/js-xpath#v0.0.2-rc1",
+    "fast-levenshtein": "^2.0.5",
+    "font-awesome": "4.7.0",
+    "grunt": "^0.4.5",
+    "grunt-cli": "^0.1.13",
+    "grunt-contrib-watch": "^0.6.1",
+    "grunt-mocha": "^0.4.13",
+    "intl-tel-input": "9.0.7",
+    "jquery": "^3.5.1",
+    "jquery-2.2.4": "npm:jquery#2.2.4",
+    "jquery-color": "2.2.0",
+    "jquery-form": "4.2.2",
+    "jquery-memoized-ajax": "^0.5.0",
+    "jquery-migrate": "1.2.1",
+    "jquery-textchange": "jmalonzo/bower-jquery-textchange#0.2.3",
+    "jquery-timeago": "rmm5t/jquery-timeago#1.2.0",
+    "jquery-tiny-pubsub": "cowboy/jquery-tiny-pubsub#~0.7.0",
+    "jquery-treetable": "^3.2.0",
+    "jquery-ui": "components/jqueryui#1.11.4",
+    "jquery.cookie": "dimagi/jquery-cookie#1.4.1",
+    "jquery.rmi": "dimagi/jquery.rmi#8c34f32817",
+    "jsdiff": "components/jsdiff#~3.2.0",
+    "knockout": "3.5.1",
+    "knockout-validation": "Knockout-Contrib/Knockout-Validation#^2.0.4",
+    "leaflet": "0.7.7",
     "less": "~3.10.3",
     "npm": "^6.14.5",
-    "uglify-js": "^2.6.1"
+    "uglify-js": "^2.6.1",
+    "markdown-it": "7.0.1",
+    "mocha": "^2.3.3",
+    "moment": "2.12.0",
+    "multiselect": "0.9.12",
+    "nprogress": "0.2.0",
+    "nvd3": "1.1.10",
+    "nvd3-1.8.6": "npm:nvd3#1.8.6",
+    "perfect-scrollbar": "^1.5.0",
+    "quicksearch": "DeuxHuitHuit/quicksearch#2.2.1",
+    "r.js": "^1.0.3",
+    "requirejs": "~2.3.1",
+    "select2": "4.0.0",
+    "topojson": "^3.0.0",
+    "ui-select": "angular-ui/ui-select#v0.13.2",
+    "underscore": "1.8.3",
+    "xpath": "dimagi/js-xpath#f4f0c78",
+    "zxcvbn": "4.2.0"
   },
   "devDependencies": {
+    "angular-mocks": "1.4.4",
     "grunt": "^0.4.5",
     "grunt-cli": "^0.1.13",
-    "chai": "^4.2.0",
+    "chai": "3.3.0",
```

Comment:
Why downgrade chai?

Reply:
it was causing certain tests to fail. Originally chai was at 3.3.0